src/ipc/ipc.cpp

```diff
@@ -1,6 +1,7 @@
 #include "ipc/ipc.hpp"
 
 #include <sys/stat.h>
+#include <sys/wait.h>
```

Comment:
Why is this needed? I don't see any wait calls in that file

Reply:
It's missing the definition for `WEXITSTATUS` (used in line 272).

Reply:
This file only has 221 lines

Reply:
Aye... this was for `lib/i3ipcpp/src/ipc.cpp`.
I'll re-create this PR after I got some much needed sleep...